httpcore/_sync/connection_pool.py

```diff
@@ -254,7 +255,7 @@ def _get_connection_from_pool(
         elif self._http2 and pending_connection is not None and not seen_http11:
             # If we have a PENDING connection, and no HTTP/1.1 connections
             # on this origin, then we can attempt to share the connection.
-            logger.trace("reusing pending connection=%r", connection)
+            logger.trace("reusing pending connection=%r", pending_connection)
```

Comment:
It looks confusing, but IDE told me that the `connection` variable "can be referenced before assignment". In fact `assert pending_connection is connection`, but `pending_connection` looks better here